SekoiaIO/activity_log/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+# meta fields (about agent gathering events)
+
+client.type:
```

Comment:
Custom fields should be prefixed with the name of the product so they are not mix with the standard ECS

Reply:
Fixed